CHANGELOG.md

```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.2.1](https://github.com/americanexpress/holocron/compare/v1.1.5...v1.2.1) (2022-02-10)
+
+
+### Bug Fixes
+
+* **externals:** mismatched externals caused one-app to fail to start ([#90](https://github.com/americanexpress/holocron/issues/90)) ([fe49f8c](https://github.com/americanexpress/holocron/commit/fe49f8cf82d7e00dd9fac925cd61d23240d92469))
```

Comment:
im not sure why this was included in the release notes

Reply:
Was the previous tag not on `main`?

Reply:
Looks like it wasn't. That would cause this

Reply:
when merging we need to make sure that we rebase and merge so that the tagged commit ends up on main